### ISSUE TITLE
decal entity crashing a game

### ISSUE THREAD

**Issue by Dziq84** [red-eclipse/base#773]
I have problems with decal entities, Using"/newent decal" in edit mode freezes game for a few seconds, after that entity is created sometimes, but in most cases it gives fatal signal 11 error

I should mention that when debuging I have no crashes when pasting or creating new decal entites, only these freezes.  /delent on a one of those ents freezes game completly, both in normal play or when debuging.

[log.txt](https://github.com/red-eclipse/base/files/1515566/log.txt)
[backtrace.txt](https://github.com/red-eclipse/base/files/1515560/backtrace.txt)

**Comment by lsalzman** [comment]
Nobody is able to independently reproduce this issue. The stack trace unfortunately does not give enough information to diagnose the problem either, so right now there is nothing that can be done.

**Comment by Dziq84** [comment]
Also I found something that may be related, but probably not.  Decals in Tesseract works great, creating a one dont affect any geometry around. In RE, creating or moving any decal ent results in geometry demip around, the same way like pushing or pulling any cube nearby

**Comment by qreeves** [comment]
Have you rebuilt before running the debugger? I can't reproduce this anymore myself.

**Comment by Dziq84** [comment]
Of course I did, build (waiting for finish), then debug

**Comment by graphitemaster** [comment]
so according to the dr memory log, this line
```
while(e.attrs[enttype[e.type].mvattr] < 0) e.attrs[enttype[e.type].mvattr] += MPV_MAX;
```

is reading one past the end of the e.attrs array

**Comment by Dziq84** [comment]
Its fixed now, everything runs ok, no more crashing, decals do not demips surrounding geometry etc :)

**Comment by shacknetisp** [comment]
Closed by b7694ae4918985bb9db96c6299f7bb87fd17f8ce.
